### PR TITLE
Add teams re-direct to homepage

### LINKED ISSUE #1043: Add hover effect and link to teams page
### What's your idea?

Given the following image, add a hover effect to the team icon similar to the hover effect on the `acmcsuf.com/teams` page on the board members. Each icon should link to the direct team on the teams page as well.

image

### PR DESCRIPTION
**Pull request by JOwen-ster**
@EthanThatOneKid @karnikaavelumani
- Added team icon re-direct on homepage
- Added OSS team to homepage
- Added scale hover effect to team icons on homepage

image

Fix #1043

### CODE REVIEW

File: src/routes/(site)/teams.svelte [hunk 1]
```diff
@@ -58,13 +58,25 @@
...
             >Level up your game development skills at the <span class="acm-red">Game Dev</span> hub</span
           >
         </div>
       </div>
+      <!-- OSS -->
+      <div class="hero-inner-container">
+        <a href="/teams#oss"><img src="/assets/oss-logo.svg" alt="OSS Team Badge" /></a>
+        <div class="size-md acm-heavy hero-text">
+          <span
+            >Take a look into the world of collaborative <span class="acm-turquoise"
```
Comment (EthanThatOneKid) [inline src/routes/(site)/teams.svelte]:
```suggestion
            >Take off into the world of collaborative <span class="acm-turquoise"
```

**Review by EthanThatOneKid — CHANGES_REQUESTED** [review]

**Review by NateGries1 — APPROVED** [review]
Game is game.

**Comment by 26samaahmed** [comment]
Looks very nice!

**Review by EthanThatOneKid — APPROVED** [review]
Thank you for optimizing the navigation experience to each team from the landing page.

**Review by k4rni — APPROVED** [review]
LGTM!

**Comment by JOwen-ster** [comment]
First contribution to site yay!

**Review by k4rni — APPROVED** [review]
LGTM! Nice work and first pr!

**Pull request merged**